packages/manager/modules/anthos/src/anthos.controller.js

```diff
@@ -0,0 +1,29 @@
+import { ANTHOS_DOCS_NAME } from './anthos.constants';
+
+export default class AnthosTenantsCtrl {
+  /* @ngInject */
+  constructor($translate, ovhDocUrl) {
+    this.$translate = $translate;
+    this.ovhDocUrl = ovhDocUrl;
+  }
+
+  $onInit() {
+    this.initGuides();
+  }
+
+  initGuides() {
+    this.guides = {};
+    this.guides.title = this.$translate.instant('anthos_tenants_guides');
+    this.guides.list = [
+      {
+        name: this.$translate.instant('anthos_tenants_guides_all'),
+        url: this.ovhDocUrl.getDocUrl(ANTHOS_DOCS_NAME),
+        external: true,
+      },
+    ];
+  }
+
+  getGuides() {
```

Comment:
This method is not used anywhere

Reply:
true, i used it in version with services.
Removed.